real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/main/RobobuggyConfigFile.java

```diff
@@ -30,20 +44,267 @@
+	 * @return
+	 */
+	public static String getCOM_PORT_IMU() {
+		return comPortImu;
+	}
+
+	/**
+	 * sets a new value to comPortImu
+	 * @param cOM_PORT_IMU
+	 */
+	public static void setCOM_PORT_IMU(String cOM_PORT_IMU) {
+		comPortImu = cOM_PORT_IMU;
+	}
+
+	/**
+	 * evaluates to the current value of configFile
+	 * @return
+	 */
+	public static String getCONFIG_FILE() {
+		return configFile;
+	}
+
+	/**
+	 * sets a new value to configFile
+	 * @param cONFIG_FILE
+	 */
+	public static void setCONFIG_FILE(String cONFIG_FILE) {
+		configFile = cONFIG_FILE;
+	}
+
+	/**
+	 * evaluates to the current value of waypointSourceLogFile
+	 * @return
+	 */
+	public static String getWAYPOINT_SOURCE_LOG_FILE() {
+		return waypointSourceLogFile;
+	}
+
+	/**
+	 * sets a new value to waypointSourceLogFile
+	 * @param wAYPOINT_SOURCE_LOG_FILE
+	 */
+	public static void setWAYPOINT_SOURCE_LOG_FILE(
+			String wAYPOINT_SOURCE_LOG_FILE) {
+		waypointSourceLogFile = wAYPOINT_SOURCE_LOG_FILE;
+	}
+
+	/**
+     * evaluates to the current value of playBackSourceFile
+	 * @return
+	 */
+	public static String getPLAY_BACK_SOURCE_FILE() {
+		return playBackSourceFile;
+	}
+
+	/**
+	 * sets a new value to playBackSourceFile
+	 * @param pLAY_BACK_SOURCE_FILE
+	 */
+	public static void setPLAY_BACK_SOURCE_FILE(String pLAY_BACK_SOURCE_FILE) {
+		playBackSourceFile = pLAY_BACK_SOURCE_FILE;
+	}
+
+	/**
+	 * evaluates to the currentValue of comPortGps
+	 * @return
+	 */
+	public static String getCOM_PORT_GPS() {
+		return comPortGps;
+	}
+
+	/**
+	 * sets a  new value to comPortGps
+	 * @param cOM_PORT_GPS
+	 */
+	public static void setCOM_PORT_GPS(String cOM_PORT_GPS) {
+		comPortGps = cOM_PORT_GPS;
+	}
+
+	/**
+	 * evaluates to the current value of gpsEnabled
+	 * @return
+	 */
+	public static boolean isGps_enabled() {
+		return gpsEnabled;
+	}
+
+	/**
+	 * sets a new value to gpsEnabled
+	 * @param gps_enabled
+	 */
+	public static void setGps_enabled(boolean gps_enabled) {
+		RobobuggyConfigFile.gpsEnabled = gps_enabled;
+	}
+
+	/**
+	 * evaluates to the current value of comPortRBSM
+	 * @return
+	 */
+	public static String getCOM_PORT_RBSM() {
+		return comPortRBSM;
+	}
+
+	/**
+	 * sets a new value to comPortRBSM
+	 * @param cOM_PORT_RBSM
+	 */
+	public static void setCOM_PORT_RBSM(String cOM_PORT_RBSM) {
+		comPortRBSM = cOM_PORT_RBSM;
+	}
+
+	/**
+	 * evaluates to the current value of encoderEnabled
+	 * @return
+	 */
+	public static boolean isEncoder_enabled() {
+		return encoderEnabled;
+	}
+
+	/**
+	 * sets a new value to encoderEnabled
+	 * @param encoder_enabled
+	 */
+	public static void setEncoder_enabled(boolean encoder_enabled) {
+		RobobuggyConfigFile.encoderEnabled = encoder_enabled;
+	}
+
+	/**
+	 * Evaluates to the current value of portVision
+	 * @return
+	 */
+	public static String getPORT_VISION() {
+		return portVision;
+	}
+
+	/**
+	 * sets a new value to portVision
+	 * @param pORT_VISION
+	 */
+	public static void setPORT_VISION(String pORT_VISION) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2254452545/issues/source?bid=2938665&fileBranchId=3180106#l289)